iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/pipe/resource/ref/PipePhantomReferenceManager.java

```diff
@@ -155,7 +155,7 @@ private void clearReferenceCount(final String holderMessage) {
       }
 
       if (referenceCount.get() >= 1) {
-        LOGGER.error("PIPE EVENT RESOURCE LEAK DETECTED: {}", holderMessage);
+        LOGGER.error("PIPE EVENT RESOURCE LEAK DETECTED ({}): {}", referenceCount, holderMessage);
```

Comment:
```suggestion
        LOGGER.error("PIPE EVENT RESOURCE LEAK DETECTED ({}): {}", referenceCount.get(), holderMessage);
```

Or you will get a non-human-readable string in ()

Reply:
no need since atomic variables override `toString` method